jaeger-core/src/main/java/io/jaegertracing/internal/exceptions/TraceIdOutOfBoundException.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2018, Uber Technologies, Inc
```

Comment:
If this is a new source file, this should be assigned to "The Jaeger Tracing Authors". See https://github.com/jaegertracing/jaeger-client-java/blob/master/CONTRIBUTING.md#license-and-certificate-of-origin

Reply:
Done